system-int-test/test/cmr/system_int_test/ingest/generics_test.clj

```diff
@@ -10,40 +10,18 @@
    [clojure.test :refer :all]
    [cmr.ingest.api.generic-documents :as gdocs]
    [cmr.system-int-test.system :as system]
+   [cmr.system-int-test.utils.generic-util :as gen-util]
    [cmr.system-int-test.utils.ingest-util :as ingest]
    [cmr.system-int-test.utils.dev-system-util :as dev-sys-util]
    [cmr.system-int-test.utils.url-helper :as url-helper]
    [cmr.transmit.config :as transmit-config])
   (:import
    [java.util UUID]))
 
-;; TODO: Generic work: known issue, a provider needs to be created through a
-;; fixture, untill this is figured out, manually create a provider before testing
-(comment use-fixtures :each
-  (join-fixtures
-   [(ingest/reset-fixture
-     {"provguid1" "PROV1"})]))
-
-;; This is the sample record that comes with Grid
-(def grid-good (-> "schemas/grid/v0.0.1/metadata.json"
-                   (jio/resource)
-                   (slurp)
-                   (json/parse-string true)))
-
-(defn generic-request
-  "This function will make a request to one of the generic URLs using the provided
-   provider and native id"
-  ([concept-type provider-id native-id] (generic-request concept-type provider-id native-id nil :get))
-  ([concept-type provider-id native-id document method]
-  (-> {:method method
-       :url (url-helper/ingest-generic-crud-url concept-type provider-id native-id)
-       :connection-manager (system/conn-mgr)
-       :body (when document (json/generate-string document))
-       :throw-exceptions false
-       :headers {"Accept" "application/json"
-                 transmit-config/token-header
-                 (transmit-config/echo-system-token)}}
-      (clj-http.client/request))))
+(use-fixtures :each (join-fixtures [(ingest/reset-fixture {"provguid1" "PROV1" "provguid2" "PROV2"}
+                                                          {:grant-all-search? false
+                                                           :grant-all-ingest? false})
+                                    (dev-sys-util/freeze-resume-time-fixture)]))
```

Comment:
why are you freezing time, I don't see any tests with time

Reply:
you're right, i put it in mostly in anticipation of the ingest tests becoming more complex down the road, but i will remove it

Reply:
removed